plone/app/contenttypes/browser/templates/folder_listing.pt

```diff
@@ -41,7 +41,7 @@
                         isAnon portal_state/anonymous;
                         normalizeString nocall: context/plone_utils/normalizeString;
                         toLocalizedTime nocall: plone_view/toLocalizedTime;
-                        show_about python:not isAnon or site_properties.allowAnonymousViewAbout;
+                        show_about python:not isAnon or context.portal_registry['plone.allow_anon_views_about'];
```

Comment:
I would also rather not use acquisition, though when we discussed with @tisto about the preferred option to access the registry in the (skins) templates, he suggested the option above.

I agree that the case here is a bit different, because we have a browser view and we also want to keep backwards compatibility. So I guess the reasoning was that if we use acquisition in a skins template is not so bad, since those templates will eventually go away anyway.

Should I change the way we access the registry in browser templates also in other packages (https://github.com/plone/Products.CMFPlone/pull/362), the way @thet did here (dropping the BBB part where it's not needed)? Or should I create a helper method in plone.app.layout?